Tests/BackendTest/ParserTest.cs

```diff
@@ -0,0 +1,80 @@
+using FluentAssertions;
+using FRITeam.Swapify.APIWrapper;
+using FRITeam.Swapify.APIWrapper.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace BackendTest
+{
+
+    public class ParserTest
+    {
+
+        [Fact]
+        public void ApiParserTestThrowException()
```

Comment:
Meaningful name could be: ParseResponce_BadInputParameter

Reply:
ok